hm11.py

```diff
@@ -0,0 +1,62 @@
+import pytz
+from datetime import datetime
+
+# Дата та час
+def func_name_and_time(func):
+    def wrapper(*args, **kwargs):
+        # eu_KY = pytz.timezone('Europe/Kyiv')
+        local = datetime.now()
+        # current_time = datetime.now(eu_KY)
+        current_time = local.strftime("%d/%m/%Y, %H:%M:%S")
+        function_name = func.__name__
+        print(f"Час виклику: {current_time} \n"
+              f"Ім'я функції: {function_name}")
+        return func(*args, **kwargs)
+    return wrapper
+
+
+@func_name_and_time
+def first_name():
+    print("Привіт, ти бачиш ім'я, час та дату виклику функції")
+
+first_name()
+
+
+print()
+
+# Custom exception is occured
+class MyCustomException(Exception):
+    def __init__(self):
+        self.message = "Custom exception is occured"
+
+try:
+    raise MyCustomException()
+except MyCustomException as a:
+    print(a.message)
+
+
+print()
+
+# Виведення 10-ти = преред і після блоку коду через контекст менеджер
+class MyContextManager:
+    def __enter__(self):
+        print("=" * 10)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("=" * 10)
+
+with MyContextManager():
+    print("Smth. code")
+
+print()
+
+# Виведення 10-ти = преред і після блоку коду через try ... except
+try:
+    print("="*10)
+    print("This is some code that will be executed.")
+except Exception as e:
+    print("An exception occurred:", e)
+else:
+    print("="*10)
+finally:
+    None
```

Comment:
You need to print 10 signs `=` here – it will allow you to have this print in any case